File: Youtube/5 soal/3.py
import pandas as pd


df = pd.read_csv('Analisis excel/Youtube/5 soal/Orders.csv')
df['Order Date'] = pd.to_datetime(df['Order Date'])
print(df)

hasil = df.groupby('Customer ID')['Order Date'].min().reset_index(name='pembelian pertama')
print(hasil)

File: Youtube/5 soal/3.r
library(dplyr)

order <- read.csv("Analisis excel/Youtube/5 soal/Orders.csv")
order$Order.Date <- as.Date(order$Order.Date, "%m/%d/%Y")
order$Ship.Date <- as.Date(order$Ship.Date, "%m/%d/%Y")
print(head(order))

hasil <- order %>% group_by(Customer.ID) %>% summarise(first_order = min(Order.Date))
print(hasil)
